previousnext/m8s
# [WIP] Filter environments list

#### What does this PR do?
Adds a search box to filter the environment list.
Fixes #58 

#### How should this be manually tested?
Use the search box, make sure the param is reflected in the url and the list is filtered accordingly

#### Any background context you want to provide?
Nope!

#### What are the relevant tickets?
https://redmine.previousnext.com.au/issues/50924

#### Screenshots (if appropriate)
![screen shot 2018-05-23 at 2 41 22 pm](https://user-images.githubusercontent.com/569927/40403784-638b2b42-5e97-11e8-82d6-b3c8eb483ac8.png)

#### Questions:
How do we make it look better?

Still needs styling/design work to improve the UX.
